api/versioninfoall.php

```diff
@@ -0,0 +1,72 @@
+<?php
+  /*
+    API documentation: https://github.com/keymanapp/keyman/wiki/downloads.keyman.com-version-API#all-versions-api
+
+    Returns a list of all versions of Keyman, starting with version 10.0, grouped by major version:
+      { "10.0": { "alpha": ["10.0.1", ...], "beta": ["10.0.111", ...], "stable": ["10.0.112", ...] }, "11.0": ..., ... }
+
+    Uses data from web folder as canonical across all platforms
```

Comment:
I suppose web is canonical from 14.0 onward.

I looked back at https://downloads.keyman.com/android/stable/ and noticed android had different versions before that.

Reply:
Good catch. I will update to that minimum for now.